packages/directory/tests/acceptance/dir-sidebar-toggle-test.ts

```diff
@@ -1,6 +1,7 @@
 import { findAll, click, currentURL, visit } from '@ember/test-helpers';
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
We could add an override in eslint to allow this in test files
https://github.com/yavin-dev/framework/blob/03be283da2f7f456af91661ebcde538749ae206b/packages/notifications/.eslintrc.js#L69